Could not find a declaration file for module 'signature_pad'

Hello,

The latest changes on #815 introduced in [v5.0.5](https://github.com/szimek/signature_pad/releases/tag/v5.0.5) trigger this error on my build:

```
error TS7016: Could not find a declaration file for module 'signature_pad'. '/path/to/project/node_modules/.pnpm/signature_pad@5.0.5/node_modules/signature_pad/dist/signature_pad.js' implicitly has an 'any' type.
  There are types at '/path/to/project/node_modules/signature_pad/dist/types/signature_pad.d.ts', but this result could not be resolved when respecting package.json "exports". The 'signature_pad' library may need to update its package.json or typings.

3 import SignaturePad from 'signature_pad';
```

I am using pnpm on node v18.20.4.

Thank you.

CC @lukescott @UziTech

--- a/package.json
+++ b/package.json
@@ -16,7 +16,8 @@
   "exports": {
     "import": "./dist/signature_pad.js",
     "require": "./dist/signature_pad.umd.js",
-    "default": "./dist/signature_pad.umd.js"
+    "default": "./dist/signature_pad.umd.js",
+    "types": "./dist/types/signature_pad.d.ts"
   },
   "scripts": {
     "build": "yarn run lint && yarn run clean && rollup --config && yarn run emit-types && yarn run update-docs",

